it/controllers/JourneyDataControllerISpec.scala

```diff
@@ -306,14 +306,15 @@ class JourneyDataControllerISpec extends ComponentSpecHelper with CustomMatchers
         stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
         val testDataKey = "testDataKey"
         val testDataValue = "testDataValue"
+        val creationTimestampKey: String = "creationTimestamp"
 
         insertById(testJourneyId, testInternalId, Json.obj(testDataKey -> testDataValue))
 
         val res = delete(s"/journey/$testJourneyId")
 
         res.status mustBe NO_CONTENT
 
-        findById(testJourneyId) mustBe Some(
+        findById(testJourneyId).map(_.-(creationTimestampKey)) mustBe Some(
```

Comment:
Can we check that the creationTimestamp field is present in the new document, something like this:

findById(testJourneyId).map(_.keys.contains(creationTimestampKey)) mustBe Some(true)